docs/extensibility/payment-method-integration.md

```diff
@@ -48,10 +50,10 @@ The options you feed the configuration instance should be an object in this shap
 
 ```js
 const options = {
-	name: 'my_payment_method',
-	content: <div>A react node</div>,
-	edit: <div> A react node </div>,
-	canMakePayment: () => true,
+  name: 'my_payment_method',
```

Comment:
Non-blocking, we could include the "details" content below inline in code comments within this example so the description is right next to the example. Might make it easier to follow.

Reply:
Ya I thought about that, the difficulty is that the details linking out won't work (because code blocks stop the link parsing in md).